src/main/java/Duke.java

```diff
@@ -1,10 +1,122 @@
+import java.util.*;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+  public static void main(String[] args) {
+    //        String logo = " ____        _        \n"
+    //                + "|  _ \\ _   _| | _____ \n"
+    //                + "| | | | | | | |/ / _ \\\n"
+    //                + "| |_| | |_| |   <  __/\n"
+    //                + "|____/ \\__,_|_|\\_\\___|\n";
+    //        System.out.println("Hello from\n" + logo);
+
+    String divider = "____________________________________________________________\n";
+    System.out.println(divider + "Hello! I'm Duke");
+    System.out.println("What can I do for you?\n" + divider);
+
+    ArrayList<Task> tasks = new ArrayList<>();
+    Scanner sc = new Scanner(System.in);
+
+    while(sc.hasNext()) {
```

Comment:
I noticed several Java reserved words such as `while` in this case not being followed by a white space. e.g.
```suggestion
    while (sc.hasNext()) {
```
Perhaps you could consider using IntelliJ's code cleanup feature to add these white spaces automatically.